src/Customers/Entry/Property.php

```diff
@@ -34,13 +34,21 @@ class Property implements Extractable, Hydratable
      */
     private $source;
 
+    /**
+     * Operation for update.
+     *
+     * @var string (replace/remove)
+     */
+    private $operation;
+
     public function hydrate(array $data, array $embedded = [])
     {
         $this->setType($data['type'] ?? null);
         $this->setSlug($data['slug'] ?? null);
         $this->setName($data['name'] ?? null);
         $this->setValue($data['value'] ?? null);
         $this->setSource($data['source'] ?? null);
+        $this->setOperation($data['operation'] ?? 'replace');
```

Comment:
While this works for this use case, I think it's a bit strange to include an `operation` here, but I understand the desire to.  I don't have any suggestions on how to improve this at the moment, I'd probably review other SDKs that support patching for inspiration.